src/main/java/com/rednavis/vaadin/showcase/jms/JmsProvider.java

```diff
@@ -0,0 +1,14 @@
+package com.rednavis.vaadin.showcase.jms;
+
+import javax.jms.ConnectionFactory;
+import org.apache.activemq.ActiveMQConnectionFactory;
+
+public class JmsProvider {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'JmsProvider' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/gh/rednavis/vaadin-showcase/pullRequest?prid=4992471)